cmd/root.go

```diff
@@ -62,13 +62,12 @@ func run(cmd *cobra.Command, args []string) {
 	profiles := readAndValidateProfile()
 	if profiles[0] == "default" && len(region) == 0 {
 		region = getDefaultAWSRegion()
-	} else {
+	} else if len(region) == 0 {
 		region = "ap-south-1"
 	}
 
 	regions := readAndValidateRegion()
-	//TODO Move this in the AWS folder
-	sess, err := config.GetSession(profiles[0], getDefaultAWSRegion())
+	sess, err := aws.GetSession(profiles[0], region)
```

Comment:
Should this not be regions[0] so we get back the validated region?

Reply:
Thanks for your input, @EDUAMEL! Although the 'region' variable is still valid, I think it would be clearer to read from the first region in the 'regions' list (regions[0]). I'll make this change, thanks again for your suggestion.